### ISSUE TITLE
Remote host module not found 'yaml'

### ISSUE THREAD

**Issue by XMLMike** [kwoodson/yedit#44]
Having an issue where a remote host is failing when running the yedit module. I'm getting the below error message:

`fatal: [local-web-server]: FAILED! => {"changed": false, "failed": true, "module_stderr": "Shared connection to 172.17.3.160 closed.\r\n", "module_stdout": "Traceback (most recent call last):\r\n  File \"/tmp/ansible_3Vz6cf/ansible_module_yedit.py\", line 191, in <module>\r\n    import yaml  # noqa: F401\r\nImportError: No module named yaml\r\n", "msg": "MODULE FAILURE", "rc": 0}`

The remote host in question is on a restricted network and as well as limited on what it can get via apt-get. It is a Debian Jessie OS. Ideally, I want to avoid using 'apt-get install python-yaml' if at all possible.

**Comment by kwoodson** [comment]
@XMLMike, 

I agree that it would be nice not to have external dependencies but that would require including a yaml parser.  I'm not sure that would be feasible as you would have to duplicate the work done in the other yaml libraries.

PyYAML or ruamel.yaml are required.

Sorry.
